internal/app/http/handler.go

```diff
@@ -231,6 +231,18 @@ func (h *TmcHandler) GetMpns(w http.ResponseWriter, r *http.Request, params serv
 	HandleJsonResponse(w, r, http.StatusOK, resp)
 }
 
+func (h *TmcHandler) GetRepos(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not 100% sure, if it should behave the same:
If e.g. a `http-repo` is not reachable,  
then `/inventory` returns `502 Bad Gateway - An upstream Thing Model repository returned an error`
but `/repos` returns an `500 Internal Server error`

Reply:
fixed